vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1518,6 +1518,17 @@ void vtkF3DRenderer::ShowConsole(bool show)
   }
 }
 
+//----------------------------------------------------------------------------
+void vtkF3DRenderer::ShowMinimalConsole(bool show)
+{
+  if (this->MinimalConsoleVisible != show)
+  {
+    this->MinimalConsoleVisible = show;
```

Comment:
you probably need to handle priority between minimal console and console. We dont want to show both :)